lib/puppet-lint/plugins/check_whitespace/right_lbrace_whitespace.rb

```diff
@@ -0,0 +1,25 @@
+# Public: Check the manifest tokens for whitespace after the left bracket that is missing and record a warning for each instance found.
+# #
+# https://docs.puppet.com/guides/style_guide.html#spacing-indentation-and-whitespace
+PuppetLint.new_check(:right_lbrace_whitespace) do
+  def check
+    tokens.each do |token|
+      next if token.next_token.nil?
+      if token.type == :LBRACE 
```

Comment:
Would it be better to look for title tokens and ensure the previous token is a single whitespace? `:LBRACE` can be found in many places, and `:WHITESPACE` could be more than a single space.